packages/workers-ai-provider/src/workersai-chat-settings.ts

```diff
@@ -10,4 +10,9 @@ export interface WorkersAIChatSettings {
 	 * @deprecated
 	 */
 	gateway?: GatewayOptions;
+
+	/**
+	 * Passthrough settings that are provided directly to the run function.
+	 */
+	[key: string]: unknown;
```

Comment:
Should this be:
```suggestion
	[key: string]: string | { toString(): string };
```
?

Reply:
Or did you want to allow for any value – but only pick out the ones that can stringify?

Reply:
> Or did you want to allow for any value – but only pick out the ones that can stringify?

I did, but I'm not sure now that you mention it. What are your thoughts?

Reply:
I think it's better to be strict here - it will give immediate feedback to the user. Will update.